packages/types/src/signUpFuture.ts

```
@@ -2,21 +2,30 @@ import type { SetActiveNavigate } from './clerk';
+interface SignUpFutureAdditionalParams {
+  firstName?: string;
+  lastName?: string;
+  unsafeMetadata?: SignUpUnsafeMetadata;
+  legalAccepted?: boolean;
+}
+
+export interface SignUpFutureCreateParams extends SignUpFutureAdditionalParams {
   transfer?: boolean;
 }
 
+export interface SignUpFutureUpdateParams extends SignUpFutureAdditionalParams {}
```

ESLint

[error] An interface declaring no members is equivalent to its supertype.

(@typescript-eslint/no-empty-object-type)